roles/kubernetes/master/templates/kubeadm-config.v1alpha1.yaml.j2

```diff
@@ -2,7 +2,8 @@ apiVersion: kubeadm.k8s.io/v1alpha1
 kind: MasterConfiguration
 api:
 {% if groups['kube-master'] | length > 1 and kubeadm_config_api_fqdn is defined %}
-  controlPlaneEndpoint: {{ kubeadm_config_api_fqdn }}:{{ loadbalancer_apiserver.port | default(kube_apiserver_port) }}
+  controlPlaneEndpoint: {{ kubeadm_config_api_fqdn }}
+  bindPort: {{ loadbalancer_apiserver.port | default(kube_apiserver_port) }}
```

Comment:
https://github.com/kubernetes/kubernetes/blob/release-1.10/cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go#L122
I think this should be correct.
/lgtm